docs/application/native/guides/location-sensors/device-sensors.md

```diff
@@ -33,9 +33,7 @@ The main features of the Sensor API include:
   >
   > Not all devices support all sensors, so each sensor is not necessarily available on all devices. You can [check whether a sensor is supported](#sensorlistener). For more information, see [System Information](../device/system.md).
 
-  **Table: Supported sensor types**
-
-  | Supported sensor types                                        |                                                  |                                                  |
+  |                                                               |                                                  |                                                  |
```

Comment:
Line 30  Replace . with :
A device can have various physical and virtual sensors. The following table lists the sensors supported by Tizen:

Reply:
I updated PR, please review again